Rendering/Core/Testing/Cxx/TestLabeledContourMapper.cxx

```diff
@@ -74,9 +77,33 @@ int TestLabeledContourMapper(int argc, char *argv[])
   tprop3->SetColor(.3, .8, .2);
   tprops->AddItem(tprop3.GetPointer());
 
```

Comment:
There are places in the mapper code that handle the case where numTProps < numContours. This exercises that functionality.